common/defs/fido/coinbase.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "Coinbase",
+  "u2f": [
+    {
+      "app_id": "e27d61b4e99de0ed98163cb38b7af933c6665e5509e8490837055813778e236a",
+      "label": "coinbase.com"
+    }
+  ]
```

I _think_ coinbase use FIDO2 not U2F:

```suggestion
  "webauthn": ["coinbase.com"],
```

---

it would make sense, per the app_id being `sha256("coinbase.com")` which is what FIDO does -- BUT the authentication runs U2F because that's what the TT screen says

i'll probably have to test it myself, but coinbase doesn't allow me to go to settings just yet

---

> BUT the authentication runs U2F because that's what the TT screen says

The question is whether TT even tries FIDO2 when there is an explicit u2f entry in the coinbase.json. Maybe the server supports both? 🤷

---

the original report also shows "U2F Register" when the id is not known yet

---

i'll just try both entries and see what happens

---

The choice of U2F vs. FIDO is up to the browser and the site. Trezor doesn't make any choice in this respect. The U2F is probably Firefox.